code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -196,6 +204,68 @@
 	else
 		return open(user)
 
+/obj/structure/closet/proc/handle_lock_addition(mob/user, obj/item/electronics/airlock/E)
+	if(!istype(E))
+		return
+	if(doinglockstuff)
+		to_chat(user, "<span class='notice'>Someone is already doing something to [src]!</span>")

```

Comment:
Sounds weird. I'd go with, like `Someone is currently working on [src]` or something.

Reply:
2lewd4u